docs/querying/api.md

```diff
@@ -132,7 +132,7 @@ URL query parameters:
 - `query=<string>`: Prometheus expression query string.
 - `start=<rfc3339 | unix_timestamp>`: Start timestamp.
 - `end=<rfc3339 | unix_timestamp>`: End timestamp.
-- `step=<duration>`: Query resolution step width.
+- `step=<duration | float>`: Query resolution step width in `duration` format or float describing number of seconds..
```

Comment:
s/describing/

Reply:
Thanks, should be fixed and amended. 
Also removed the double trailing dot.